### DISCUSSION TITLE
Payment Providers

### DISCUSSION THREAD

**Discussion by nico-guba** [nico-guba/deloitte-advantage#3] [Ideas]
We would need to collect a list of payment providers.

Ideally they should have a sandbox environment where one can make fake payments for testing purposes.

There might also be an issue of having reasonable charges.   We also need to think about recurring payments and whether its best to leave it up to a 3rd party or whether we have our own system implement this feature.   Naturally, having advantage perform this would give us more control and possibly better reporting.

Please add your findings to the list below.  

- [Adyen](https://www.adyen.com/en_GB/)
- [Stripe](https://stripe.com/gb)
- [Worldpay](https://online.worldpay.com/)
- [Square](https://developer.squareup.com/gb/en/online-payment-apis)

**Comment by mchadwickDD** [comment]
Worldpay Pricing information:
- https://online.worldpay.com/pricing
- https://www.pay360.com/products-and-services/worldpaymerchants

Worldpay implementation detail:
- https://developerengine.fisglobal.com/apis/ips/integration/direct-integration

**Comment by mchadwickDD** [comment]
Stripe Pricing Information:
- https://stripe.com/gb/pricing

Stripe Implementation detail:
- https://stripe.com/docs/payments/accept-a-payment

**Comment by mchadwickDD** [comment]
Adyen pricing information
- https://www.adyen.com/en_GB/pricing

Adyen implementation detail:
- https://docs.adyen.com/online-payments

**Comment by mchadwickDD** [comment]
Square pricing information:
- https://developer.squareup.com/gb/en/online-payment-apis

Square implementation detail:
- https://developer.squareup.com/reference/square
